my_project/tab_natural_ventilation/app_natural_ventilation.py

```diff
@@ -202,7 +203,7 @@ def inputs_tab():
                     dbc.Checklist(
                         options=[
                             {
-                                "label": "Avoid condensation with radiant systems: If the outdoor dew point temperature is below the radiant system surface temperature, the data point is not plot.",
+                                "label": "Avoid condensation with radiant systems: If the outdoor dew point temperature is below the radiant system surface temperature, the data point is not plot. -- (default: °C) / °F",
```

Comment:
same as the previous comment. This should not be hardcoded.

Reply:
This problem is now solved. The set number will be changed automatically according to the si-ip-state.